### PR TITLE
Refresh concepts/entity-template 

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description

refresh the `GDK concepts - entity templates` page

#### Tests
improbadoc serve

#### Documentation
this is the documentation

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/reference/concepts/entity-templates.md [hunk 1]
```diff
@@ -1,67 +1,62 @@
...
+  * [MonoBehaviour and ECS workflows]({{urlRoot}}/reference/workflows/which-workflow)
+  * [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+  * [Standard schema library components](https://docs.improbable.io/reference/latest/shared/glossary#standard-schema-library-components)
+")%>
 
-For information on how to create SpatialOS entities once you have set up entity templates, see the documentation on [creating entities (MonoBehaviour workflow)]({{urlRoot}}/reference/workflows/monobehaviour/interaction/commands/create-delete-spatialos-entities) and [ECS World commands (ECS workflow)]({{urlRoot}}/reference/workflows/ecs/interaction/commands/world-commands).
+An [`EntityTemplate`]({{urlRoot}}/api/core/entity-template) specifies what [components]({{urlRoot}}/reference/glossary#spatialos-component) a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) will have, the initial values of these components and which [layers](https://docs.improbable.io/reference/latest/shared/glossary#layers) get [write access]({{urlRoot}}/reference//glossary#authority) to each component.
```
Comment (jessicafalk) [inline docs/reference/concepts/entity-templates.md]:
```suggestion
An [`EntityTemplate`]({{urlRoot}}/api/core/entity-template) specifies what [components]({{urlRoot}}/reference/glossary#spatialos-component) a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) contains, the initial values of these components and which [layers](https://docs.improbable.io/reference/latest/shared/glossary#layers) get [write access]({{urlRoot}}/reference//glossary#authority) to each component.
```

File: docs/reference/concepts/entity-templates.md [hunk 2]
```diff
@@ -1,67 +1,62 @@
...
-For each [schema component]({{urlRoot}}/reference/glossary#schema) you define, the `code generator` creates a struct which inherits from `ISpatialComponentSnapshot`, the generated [snapshot struct]({{urlRoot}}/reference/concepts/code-generation#snapshot). For example, for the following schema:
+All SpatialOS entities require the `Position` and `EntityAcl` components.
 
-```
-component Health {
-  id = 1;
-  int32 current_health = 1;
-}
-```
+The `Position` component must be added to the entity template manually. It is used by SpatialOS for load-balancing purposes and some types of queries.
```
Comment (jessicafalk) [inline docs/reference/concepts/entity-templates.md]: do we want to be more specific about what kind of queries?

File: docs/reference/concepts/entity-templates.md [hunk 3]
```diff
@@ -1,67 +1,62 @@
...
+
+The `EntityAcl` component is automatically handled by the `EntityTemplate` class. It is this component that determines which types of workers have read access to an entity and, for each component, which type of worker can have write access.
```
Comment (jessicafalk) [inline docs/reference/concepts/entity-templates.md]:
```suggestion
The `EntityAcl` component is automatically handled by the `EntityTemplate` class. This component determines which types of workers have read access to an entity and, for each component, which type of worker can have write access.
```

should we add a note that only one worker can be authoritative?

File: docs/reference/concepts/entity-templates.md [hunk 4]
```diff
@@ -1,67 +1,62 @@
...
 
-The code generator creates:
+> Understand more about the standard schema library components [here](https://docs.improbable.io/reference/latest/shared/glossary#standard-schema-library-components).
```
Comment (jessicafalk) [inline docs/reference/concepts/entity-templates.md]:
```suggestion
> To learn more about the standard schema library components, [go to this documentation](https://docs.improbable.io/reference/latest/shared/glossary#standard-schema-library-components).
```

File: docs/reference/concepts/entity-templates.md [hunk 5]
```diff
@@ -1,67 +1,62 @@
...
+
+To add components to the entity template, use the `AddComponent` method. For example, to add a `Position` component with [write access]({{urlRoot}}/reference//glossary#authority) given to UnityGameLogic workers:
```
Comment (jessicafalk) [inline docs/reference/concepts/entity-templates.md]:
```suggestion
To add components to the entity template and specify which worker type has write access over the component, use the `AddComponent` method.
```

File: docs/reference/concepts/entity-templates.md [hunk 6]
```diff
@@ -1,67 +1,62 @@
...
 
 ```csharp
-public partial class Health
-{
-    public struct Snapshot
-    {
-        int CurrentHealth;
-    }
-}
+entityTemplate.AddComponent(new Position.Snapshot(coords), "UnityGameLogic");
```
Comment (jessicafalk) [inline docs/reference/concepts/entity-templates.md]:
```suggestion
// Adds the Position component to the template and specifies that only a UnityGameLogic worker can have write access.
entityTemplate.AddComponent(new Position.Snapshot(coords), "UnityGameLogic");
```

File: docs/reference/concepts/entity-templates.md (hunk 2 above)
Comment (paulbalaji) [reply docs/reference/concepts/entity-templates.md]: Ah I forgot to link this to a couple of other docs pages

I think we could give an example e.g. relative constraints

File: docs/reference/concepts/entity-templates.md (hunk 6 above)
Comment (paulbalaji) [reply docs/reference/concepts/entity-templates.md]: updated

File: docs/reference/concepts/entity-templates.md [hunk 7]
```diff
@@ -1,67 +1,64 @@
...
+An [`EntityTemplate`]({{urlRoot}}/api/core/entity-template) specifies what [components]({{urlRoot}}/reference/glossary#spatialos-component) a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) contains, the initial values of these components and which [layers](https://docs.improbable.io/reference/latest/shared/glossary#layers) get [write access]({{urlRoot}}/reference//glossary#authority) to each component.
 
-## How to create a SpatialOS entity template
+## How to define an entity template
 
-You have to create an `EntityTemplate` to specify which components a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) has and the initial values of those [components]({{urlRoot}}/reference/glossary#spatialos-component). You have to also specify which type of workers can have write access (also known as ”[authority]({{urlRoot}}/reference/glossary#authority)”) on a per-component basis.
+To define an entity template, you must first construct an `EntityTemplate`.
```
Comment (jamiebrynes7) [inline docs/reference/concepts/entity-templates.md]: O rly

**Pull request merged**